src/util.ts

```diff
@@ -359,7 +359,7 @@ export function* flatMap<In, Out>(rng: Iterable<In>, fn: (item: In) => Iterable<
   }
 }
 
-export function splitEnvironmentVars(env: EnvironmentVariables): EnvironmentVariables[] {
+export function expandEnvironmentVars(env: EnvironmentVariables): EnvironmentVariables[] {
```

Comment:
I would suggest to change this back to split. Term 'expand' may cause confusion with terms used in expand.ts.